mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -3449,3 +3450,55 @@ fn test_match_utxo_with_eth_taker_buy() {
     block_on(mm_bob.stop()).unwrap();
     block_on(mm_alice.stop()).unwrap();
 }
+
+#[test]
+fn test_locked_amount() {
+    let (_ctx, _, bob_priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN", 1000.into());
+    let (_ctx, _, alice_priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN1", 1000.into());
+    let coins = json!([mycoin_conf(1000), mycoin1_conf(1000)]);
+    let bob_conf = Mm2TestConf::seednode(&format!("0x{}", hex::encode(bob_priv_key)), &coins);
+    let mut mm_bob = MarketMakerIt::start(bob_conf.conf, bob_conf.rpc_password, None).unwrap();
+    let (_bob_dump_log, _bob_dump_dashboard) = mm_dump(&mm_bob.log_path);
+
+    let alice_conf = Mm2TestConf::light_node(&format!("0x{}", hex::encode(alice_priv_key)), &coins, &[&mm_bob
+        .ip
+        .to_string()]);
+    let mut mm_alice = MarketMakerIt::start(alice_conf.conf, alice_conf.rpc_password, None).unwrap();
+    let (_alice_dump_log, _alice_dump_dashboard) = mm_dump(&mm_alice.log_path);
+
+    log!("{:?}", block_on(enable_native(&mm_bob, "MYCOIN", &[])));
+    log!("{:?}", block_on(enable_native(&mm_bob, "MYCOIN1", &[])));
+    log!("{:?}", block_on(enable_native(&mm_alice, "MYCOIN", &[])));
+    log!("{:?}", block_on(enable_native(&mm_alice, "MYCOIN1", &[])));
+
+    block_on(start_swaps(
+        &mut mm_bob,
+        &mut mm_alice,
+        &[("MYCOIN", "MYCOIN1")],
+        1.,
+        1.,
+        777.,
+    ));
+
+    let locked_bob = block_on(get_locked_amount(&mm_bob, "MYCOIN"));
+    assert_eq!(locked_bob.coin, "MYCOIN");
+
+    let expected_num = MmNumber::from("777.00001");
+    let expected_result = MmNumberMultiRepr {
```

Comment:
`structs::MmNumberMultiRepr` implements `From<T> where MmNumber: From<T>`, so it can be simplified as
```rust
let expected_result = MmNumberMultiRepr::from("777.00001");
```